translations/es.json

```
@@ -60,23 +60,23 @@
-	"Alternatively you could install one of these drivers:": "",
+	"Alternatively you could install one of these drivers:": "Alternativamente, puede installar uno de estos drivers:",
 	"Amount of RAM allocated to the minikube VM (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g)": "Cantidad de RAM asignada a la VM de minikube (formato: \u003cnúmero\u003e[\u003cunidad\u003e], donde unidad = b, k, m o g)",
 	"Amount of time to wait for a service in seconds": "Cantidad de tiempo para esperar por un servicio en segundos",
 	"Amount of time to wait for service in seconds": "Cantidad de tiempo para esperar un servicio en segundos",
 	"Another hypervisor, such as VirtualBox, is conflicting with KVM. Please stop the other hypervisor, or use --driver to switch to it.": "Otro hipervisor, por ejemplo VirtualBox, está en conflicto con KVM. Por favor detén el otro hipervisor, o usa --driver para cambiarlo.",
-	"Another minikube instance is downloading dependencies... ": "",
+	"Another minikube instance is downloading dependencies... ": "Otra instancia de minikube esta descargando dependencias...",
 	"Another program is using a file required by minikube. If you are using Hyper-V, try stopping the minikube VM from within the Hyper-V manager": "Otro programa está usando un archivo requerido por minikube. Si estas usando Hyper-V, intenta detener la máquina virtual de minikube desde el administrador de Hyper-V",
-	"At least needs control plane nodes to enable addon": "",
+	"At least needs control plane nodes to enable addon": "Al menos se necesita un nodo de plano de control para habilitar el addon",
 	"Automatically selected the {{.driver}} driver": "Controlador {{.driver}} seleccionado automáticamente",
 	"Automatically selected the {{.driver}} driver. Other choices: {{.alternates}}": "Controlador {{.driver}} seleccionado automáticamente. Otras opciones: {{.alternates}}",
 	"Available Commands": "Comandos disponibles",
 	"Basic Commands:": "Comandos basicos:",
 	"Because you are using a Docker driver on {{.operating_system}}, the terminal needs to be open to run it.": "Porque estás usando controlador Docker en {{.operating_system}}, la terminal debe abrirse para ejecutarlo.",
 	"Bind Address: {{.Address}}": "Dirección de enlace: {{.Address}}",
-	"Booting up control plane ...": "",
+	"Booting up control plane ...": "Iniciando plano de control",
```

```suggestion
	"Booting up control plane ...": "Iniciando plano de control ...",
```
We should keep the `...` because a spinner comes up and would look a bit weird next to the text.